e2e-tests/react-client/app/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@fishjam-dev/react-client-e2e",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "license": "Apache-2.0",
+  "scripts": {
+    "e2e": "NODE_OPTIONS=--dns-result-order=ipv4first playwright test",
```

Comment:
why is `--dns-result-order=ipv4first` used?

Reply:
Because of Colima:
https://node.testcontainers.org/supported-container-runtimes/